stock_exchange/apps/stock/models.py

```diff
@@ -1,4 +1,77 @@
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from datetime import date
 
+from stock_exchange.apps.users.models import User
 
-# Add your models here
\ No newline at end of file
+#This tuple is used for choice option for
```

Comment:
Comments in python or any other language when commenting with # should have space between # and text
# This tuple... + english